spec/Section 7 -- Response.md

```diff
@@ -135,14 +145,103 @@ locations, where each location is a map with the keys `line` and `column`, both
 positive numbers starting from `1` which describe the beginning of an
 associated syntax element.
 
+If an error can be associated to a particular field in the GraphQL result, it
+should contain an entry with the key `path` that details the path of the
```

Comment:
I'm with @helfer on this one, especially the argument that we could always loosen in the future but not as easily tighten. "Should" is essentially a strong recommendation to do something but provides an opt out should there be a good reason, though I'm having trouble thinking of what a reason would be.

@aweiker yes though I'm not sure this should be a challenge given that the response path is data that already needs to be known in order to appropriate build a serialized JSON value. Right now most implementations probably implicitly know this by the nature of recursing to execute all fields, but some information about the ancestor fields can be kept pretty inexpensively, as has already been showed in the reference implementation.

Every *field error* (during execution) has a path, but *query errors* (e.g. all validation errors) do not have a path.